rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java

```diff
@@ -462,89 +462,89 @@ public void registerAsListener() throws NotConnectedException, PermissionExcepti
 
     @Override
     public void setSession(String sid) {
-        ((ISchedulerClient) _getScheduler()).setSession(sid);
+        _getScheduler().setSession(sid);
     }
 
     @Override
     public String getSession() {
-        return ((ISchedulerClient) _getScheduler()).getSession();
+        return _getScheduler().getSession();
     }
 
     @Override
     public boolean isJobFinished(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
-        return ((ISchedulerClient) _getScheduler()).isJobFinished(jobId);
+        return _getScheduler().isJobFinished(jobId);
     }
 
     @Override
     public boolean isJobFinished(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
-        return ((ISchedulerClient) _getScheduler()).isJobFinished(jobId);
+        return _getScheduler().isJobFinished(jobId);
     }
 
     @Override
     public JobResult waitForJob(JobId jobId, long timeout)
             throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)